src/services/codefixes/convertToAsyncFunction.ts

```diff
@@ -156,11 +160,8 @@ namespace ts.codefix {
                 forEachChild(node, visit);
                 return;
             }
-
             const symbol = checker.getSymbolAtLocation(node);
-            const isDefinedInFile = symbol && declaredInFile(symbol, sourceFile);
-
-            if (symbol && isDefinedInFile) {
+            if (symbol && (declaredInFile(symbol, sourceFile) || isImported(symbol))) {

```

Comment:
if `symbol` is defined, what other options are there besides imported or declared-in-file? Can we get by with just `if (symbol)` ?

Reply:
Good question. I thought about that option, however, decided to extend the condition with `isImported`, because `declaredInFile` was added for some reasons, maybe to exclude global declarations - not sure.

Reply:
If it is actually excluding things like global declarations, let's leave it, but otherwise, seems fine to take out.

Reply:
I can try to leave `if (symbol)`, if all tests will work I push changes, otherwise will leave these changes as is. @sandersn  
@uniqueiniquity Is it ok?

Reply:
I'm fine with it, but I'll defer to @sandersn here.

Reply:
I've checked and all tests (`convertToAsyncFunction`)work fine. @sandersn

Reply:
I've pushed changes without `declaredInFile`, `isImported`. @sandersn if you think the previous changes are better, I can return them.